blazor/data-form/templates.md

```diff
@@ -0,0 +1,262 @@
+---
+layout: post
+title: Templates in Blazor DataForm Component | Syncfusion
+description: Checkout and learn here about how to customize the specific editor component or entire form components in Blazor DataForm component.
+platform: Blazor
+control: DataForm
+documentation: ug
+---
+
+# Templates in DataForm component
+
+In DataForm component we can customize the specific editor field or the entire form components using template feature. 
+
+## Customization of specific field editor
+
+We can customize the particular field editor with required UI customization using the [Template](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DataForm.Template.html) `RenderFragment` inside `FormItem` tag.
+
+{% tabs %}
+{% highlight razor tabtitle="Template" hl_lines="3 10" %}
+
+@using Syncfusion.Blazor.DataForm
+@using System.ComponentModel.DataAnnotations
+@using Syncfusion.Blazor.Inputs
+@using Syncfusion.Blazor.Calendars
+
+
+<SfDataForm ID="MyForm"
+            Model="@CreditCardModel"
+            Width="50%">
+            
+    <FormValidator>
+        <DataAnnotationsValidator></DataAnnotationsValidator>
+    </FormValidator>
+
+    <FormItems>
+        <FormItem Field="@nameof(CreditCardModel.Name)" Placeholder="e.g. Andrew Fuller" LabelText="Name on card" ></FormItem>
+        <FormItem Field="@nameof(CreditCardModel.CardNumber)" LabelText="Card Number" >
+        </FormItem>
+        <FormItem Field="@nameof(CreditCardModel.CVV)">
+            <Template>
+                <label class="e-form-label">CVV*:</label>
+                <SfMaskedTextBox Mask="000" @bind-Value="CreditCardModel.CVV" ></SfMaskedTextBox>
+            </Template>
+        </FormItem>
+        <FormItem Field="@nameof(CreditCardModel.ExpiryDate)">
+            <Template>
+                <label class="e-form-label">Expiry Date*:</label>
+                <SfDatePicker TValue="DateTime?" Format="MM/yy" EnableMask="true"></SfDatePicker>
+            </Template>
+        </FormItem>
+    </FormItems>
+
+</SfDataForm>
+
+
+@code {
+    public char PromptCharacter { get; set; } = ' ';
+
+    public class CreditCard
+    {
+        [Required(ErrorMessage = "Please enter the name on card")]
+        public string Name { get; set; }
+
+        [Required(ErrorMessage = "Please enter the card number")]
+        [CreditCard]
+        public string CardNumber { get; set; }
+
+        [Required(ErrorMessage = "Please enter cvv number")]
+        public string CVV { get; set; }
+
+        [Required(ErrorMessage = "Please select/enter expiry date")]
+        public DateTime? ExpiryDate { get; set; }
+    }
+
+    private CreditCard CreditCardModel = new CreditCard();
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+![Blazor DataForm Form Item](images/blazor_dataform_template.png)
```

Comment:
Include reference to the form items sections and give details about using Form items - Template in combination with editType and autogenerateItems